src/apps/explorer/pages/Home/index.tsx

```diff
@@ -1,43 +1,33 @@
 import React from 'react'
 import styled from 'styled-components'
-import { media } from 'theme/styles/media'
-
-import homeTeaser from './home-teaser.png'
+import { Search } from 'apps/explorer/components/common/Search'
 
 const Wrapper = styled.div`
   display: flex;
   justify-content: center;
+  flex-flow: column wrap;
+  height: calc(100vh - 15rem);
```

Comment:
Should we have a page component, or some kind of container that do this? I mean, there's many places where we have the issue of the footer jumping to the top.

Also should it be min-height or height?

Reply:
For the homepage doing a height, to have somewhat of a fixed height layout, where we keep in mind the footer. Many ways about it.

As for a page component: In my opinion, when you say the 'footer jumps' that is an effect of a page without content and/or flash of content. We therefore should solve that in individual cases: A normal page with at least a minimum amount of content never will have this problem.

On the order detail page the cause of the footer to perceivably 'jump' is because initially it is loading the order. We should address that by setting a minimum container height. Or even better, render the full table with all (minimum) columns but in a loading state for the values. 

Again, think we should address that separately.

Reply:
I'm referring to #285